lib/invitations/tests/fixtures/Template.ts

```diff
@@ -1,3 +1,5 @@
+import { NotificationsInterval, NotificationsIntervalDefault } from "packages/types/src/form-types";
```

Comment:
This should reference the gcforms package import.
```suggestion
import { NotificationsInterval, NotificationsIntervalDefault } from "@gcforms/types";
```